tools/update_helmchart.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/sh
```

Comment:
I don't think it helps aiming for POSIX compatibility in this script, since it is meant to run in an environment under our control. I think it just hurts readability.

Also, regarding version parsing:
- Maybe we shouldn't reinvent the wheel and instead use something like this: https://github.com/cloudflare/semver_bash, then again maybe it's not a good idea to rely on external tools for this. I don't know.
- It would have been simpler if you could just pass the new version as an argument. I am not sure in which situation you would want this script to calculate the new version by itself.

Reply:
> I don't think it helps aiming for POSIX compatibility in this script, since it is meant to run in an environment under our control. I think it just hurts readability.

Yeah whilst ai would normally agree with you; I don't really want to pull in an external dep here. I also based this off of existing code we wrote for the NetData Installer but I'm improving it overall so we can use it in the Static installer too -- It _has_ to be POSIX or things don't really work out for folks on smaller/embedded systems without Bash.

> Also, regarding version parsing:
> 
> * Maybe we shouldn't reinvent the wheel and instead use something like this: https://github.com/cloudflare/semver_bash, then again maybe it's not a good idea to rely on external tools for this. I don't know.

See comment above :)

> * It would have been simpler if you could just pass the new version as an argument. I am not sure in which situation you would want this script to calculate the new version by itself.

I can add this capability easily. Normally we just bump the version though and this is useful to maintain the `version` of the helm-chart itself in lock-step. I will add the capability of passing in an arbitrary value for `appVersion` though in a new commit.

Reply:
> I will add the capability of passing in an arbitrary value for appVersion though in a new commit.

Done

Reply:
> I also based this off of existing code we wrote for the NetData Installer but I'm improving it overall so we can use it in the Static installer too

If it's based on existing code, :+1: for reusability.